src/core/config/descriptions.h

```diff
@@ -70,4 +70,10 @@ constexpr auto kDWeight =
 constexpr auto kDBumpsLimit = "max considered intervals amount. Pass 0 to remove limit";
 constexpr auto kDIterationsLimit = "limit for iterations of sampling";
 constexpr auto kDACSeed = "seed, needed for choosing a data sample";
+constexpr auto kDTableSet =
+        "A set of tables constituting some dataset, which will be "
+        "processed by an algorithm";
+constexpr auto kDHllAccuracy = "(Faida special) HyperLogLog approximation accuracy";
+constexpr auto kDSampleSize = "(Faida special) Size of a table sample";
+constexpr auto kDFindNary = "(Faida special) Detect n-ary inclusion dependencies [true|false]";
```

Comment:
In the case of functional dependencies in the desbordante, it is possible to explicitly set a limit on the number of attributes on the left side of the dependency (`--max_lhs`). Is there any point and opportunity to do something similar here? This will be more consistent (relative to existing code) and convenient (for the user)

Reply:
Thanks for the answer, I like the suggested name for the option.
It would be good to open an issue if we don’t want to implement it as part of this pr

Reply:
Created an issue: https://github.com/Mstrutov/Desbordante/issues/302